Tests/loginTests.js

```diff
@@ -0,0 +1,83 @@
+var homePage = require('../Pages/homePage.js'),
+    loginPage = require('../Pages/loginPage'),
+    itemPage = require ("../Pages/itemPage"),
+    registerPage = require ("../Pages/registerPage"),
+    data = require("../Data/data.js");
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("003: Login", function(){
+    
+    beforeEach(() => {
+        homePage.openPageURL(data.homepageLink);
+        })
+    afterEach(() => {
```

Comment:
Please move "afterEach" block to the bottom

Reply:
I moved it down to the bottom in test classes.